meta_mixin/models.py

```diff
@@ -2,6 +2,7 @@
 from copy import copy
 
 from django.contrib.sites.models import Site
+from meta.settings import SITE_PROTOCOL
 
```

Comment:
You must pick this setting from ``django.conf.settings`` instead than from ``meta.settings``, as the latter is not updated by ``override_settings``